documentation/modules/rn-2.6.adoc

```diff
@@ -2,6 +2,293 @@
+
+.Left over `ovirtvolumepopulator` from failed migration causes plan to stop indefinitely in `CopyDisks` phase.
+
+An earlier failed migration can leave an outdated `ovirtvolumepopulator` in the project of a new plan for the same VM. The `CreateDataVolumes` phase does not create populator PVCs when transitioning to `CopyDisks`, causing the `CopyDisks` phase to stay indefinitely. link:https://issues.redhat.com/browse/MTV-929[(MTV-929)]
+
+.Unclear error message when Forklift fails to build/create a PVC
+
+The migration fails to build the PVC when the destination storage class does not have a configured storage profile. The error logs lack clear information to identify the reason for the failure. link:https://issues.redhat.com/browse/MTV-928[(MTV-928)]
+
+.Fail to migrate a VM with NVME disks from vSphere 
+
+When migrating a VM with NVME disks from vSphere, the migration process fails and the Web Console shows that the `Convert image to kubevirt` stage is `running` but did not finish successfully. link:https://issues.redhat.com/browse/MTV-963[(MTV-963)]
+
+.Import image-based VMs
+
+Migrating an image-based VM without a `virtual_size` field can fail on a block mode storage class. link:https://issues.redhat.com/browse/MTV-946[(MTV-644)]
+
+For a complete list of all known issues in this release, see the list of link:https://issues.redhat.com/issues/?filter=12430275[Known Issues] in Jira. 
+
+[id="resolved-issues-26_{context}"]
+== Resolved issues
+
+This release has the following resolved issues:
+
+// Has this been resolved?
+.Possible data loss when migrating VMware VMs with snapshots
+
+In previous versions of {product-short}, when migrating a VMware VM that had a snapshot, the VM that was created in {ocp-name} Virtualization contained the data in the snapshot, but not in the running VM that was migrated. This has been resolved in {product-short} 2.6.0. [(MTV-447)]
+
+.Adding an OVA provider raises a `ConnectionTestFailed` error message
+
+In previous versions of {product-short}, when adding an OVA provider, showing the error message `The provider is not ready - ConnectionTestFailed`, although it eventually creates the provider. The message disappears after a few seconds. This has been resolved in {product-short} 2.6.0. link:https://issues.redhat.com/browse/MTV-671[(MTV-671)]
+
+.Canceling and deleting a failed migration plan does not clean up the `populate` pods and PVC
+
+When a user cancels and deletes a failed migration plan after creating a PVC and spawning the `populate` pods, the `populate` pods and PVC are not deleted. You must delete the pods and PVC manually. link:https://issues.redhat.com/browse/MTV-678[(MTV-678)]
```

Comment:
This one is documented as a Known Issue. Shall this be moved to the Known issues section or the text must be just rewritten as the Resolved issue?

Reply:
yeah, this needs to be documented as a resolved issue